json-rpc/Cargo.toml

```diff
@@ -44,7 +44,10 @@ proptest = "0.10.1"
 reqwest = { version = "0.10.8", features = ["blocking", "json"], default_features = false }
 rand = { version = "0.7.3" }
 
+config-builder = { path = "../config/config-builder", version = "0.1.0" }
```

Comment:
so I guess you don't really need this dependency?

Reply:
it is required for BuildSwarm trait implemented by ValidatorBuilder#build_swarm